test/gypbuild-test.js

```diff
@@ -43,6 +44,8 @@ test('gyp is invoked with correct arguments, release mode', function (t) {
     log: noop
   }
   runGyp(opts, 'x.y.z', function (err) {
+    var devDir = opts.gyp.devDir
+    t.equal(devDir.indexOf(osenv.home()), 0, devDir + ' should start with home folder')
```

Comment:
It's a little bit silly sometimes when you write test code that's basically the same as the code you wrote.